ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -716,16 +764,12 @@ func (pg *DEXOnboarding) stepWaitForBondConfirmation(gtx C) D {
 					)
 				}),
 				layout.Rigid(func(gtx C) D {
-					return layout.Inset{Top: 10, Bottom: u10}.Layout(gtx, pg.Theme.Body1(values.StringF(values.StrDEXBondConfirmationMsg, "dex.decred.org", 2 /* TODO: use real values */)).Layout)
+					return layout.Inset{Top: 10, Bottom: u10}.Layout(gtx, pg.Theme.Body1(values.StringF(values.StrDEXBondConfirmationMsg, pg.selectedServerURL, pg.requiredBondConf)).Layout)
```

Comment:
Is it possible for each step to have a state struct that holds all the variables for that step? I think that would be easier to follow than `pg` having many variables for different steps.